app/main.py

```diff
@@ -4,5 +1,50 @@
+class Car():
+
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+
+        washing_price = (car.comfort_class
+                         * (self.clean_power - car.clean_mark)
+                         * self.average_rating
+                         / self.distance_from_city_center)
+        return round(washing_price, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rate: int) -> None:
+
+        sum_of_ratings = self.average_rating * self.count_of_ratings
+        self.count_of_ratings += 1
+        self.average_rating = round((sum_of_ratings + rate)
+                                    / self.count_of_ratings, 1)
+
+    def serve_cars(self, cars: list) -> float:
+
+        income = 0
+
+        for car in cars:
+
+            # Obliczenie przychodu
+            if car.clean_mark < self.clean_power:
```

Comment:
The condition here checks if the car's `clean_mark` is less than `clean_power`, which is correct for determining if the car needs washing. However, ensure that this logic aligns with the intended behavior of the `serve_cars` method, which should only wash cars that are not clean enough.